### ISSUE TITLE
CI: conda 1.9 and 1.10 getting killed

### ISSUE THREAD

**Issue by akihironitta** [PyTorchLightning/pytorch-lightning#10129]
## 🐛 Bug
`conda (3.7, 1.9)` and `conda (3.7, 1.10)` runs non-deterministically fail with the error:

```
tests/helpers/test_models.py::test_models[None-ParityModuleRNN] PASSED   [ 31%]
/__w/_temp/2a0a0b32-902c-4e18-bf17-5c09efcb74cf.sh: line 2:    90 Killed                  coverage run --source pytorch_lightning -m pytest pytorch_lightning tests -v --durations=50 --junitxml=junit/test-results-Linux-torch1.10.xml
tests/helpers/test_models.py::test_models[None-ParityModuleMNIST] 
Error: Process completed with exit code 137.
```

### To Reproduce
See `conda (3.7, 1.9)` and `conda (3.7, 1.10)`:
- Successful runs: https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml?query=is%3Asuccess
- Failing runs: https://github.com/PyTorchLightning/pytorch-lightning/actions/workflows/ci_test-conda.yml?query=is%3Afailure

### Expected behavior
Not getting killed.

### Additional context
[closed] First try: #10077
[merged] Temporary fix: #10084

**Comment by Borda** [comment]
@akihironitta still happening?

**Comment by akihironitta** [comment]
It seems the fix #10084 is working fine :)
